hugo/README.md

```diff
@@ -67,6 +67,16 @@ If all above is done and the icon is added in the right folder you can run the f
 npm run icons
 ```
 
+### Algolia
+
+If you want to test Algolia local, you can add the environoment variables from Algolia to the environment config:
+
+```
+cp .env.sample .env
+```
+
+The values can be found in Algolia. For testing, create a new index. DO NOT USE THE cuelang.org INDEX.
```

Comment:
Maybe we can allready create a test index for this that can be used. You have a maximum of index so we don't want everybody to just create new indexes

Reply:
How is Algolia tested locally?

> You have a maximum of index so we don't want everybody to just create new indexes

What does this mean?

Reply:
Maybe we should just refrase this sentence. If you want to test something with algolia you don't want to overwrite the current index because that would mess up search results on alpha.So if we need to add something for instance and we need to test is we have to create a test index to play around with it without changing the production index. So we could say that a develop who needs to test something with algolia has to create a new index just for testing and after they're done they should remove it or something so we don't end up with 'dead' test indexes

Reply:
Reworded this into a roadmap